lib/src/time_converters.dart

```diff
@@ -0,0 +1,12 @@
+const _microsecFromUnixEpochToY2K = 946684800 * 1000000;
```

Comment:
Nitpicking: This seems to be a magic number. Can we get the same value from `DateTime.utc([...?]).microsecondsSinceEpoch` ?

Reply:
Yeah I don't know what I was thinking.. fixed!